frontend/src/components/pages/Kunnskapkom.tsx

```diff
@@ -0,0 +1,50 @@
+import clsx from "clsx";
+import { useState } from "react";
+
+const backgroundImageStyling = {
+  backgroundImage: "url('https://creazilla-store.fra1.digitaloceanspaces.com/cliparts/820035/bookshelf-clipart-xl.png')",
+  backgroundPosition: "bottom -10px left -10px",
+  backgroundSize: "30%",
+  backgroundRepeat: "no-repeat",
+};
+
+const funFactList = [
+  ["Forelesning: ", "[foːrɛleːsniːŋ] Et teoretisk konsept om å tilegne seg kunnskap i forelesningssal. Utøves ikke i praksis."],
+  ["Semesterkrise: ", "[seːmæsteːrkriːsæ] Den eksistensielle krisen som inntreffer når man innser at eksamen er nærmere enn forventet."],
+  ["Stack overflow: ", "[stæk oʊvɚfloʊ] Eldgammelt forum for programmeringsdiskusjoner. Blir fortsatt besøkt i ny og ne når ChatGPT glemmer forskjellen på Python2 og Python3."],
+  ["Programmeringsbrus: ", "[prugrɑmeːriːŋsbrʉs] Syntetisk væske som gir en illusjon av forbedrede programmeringsferdigheter. Selges for 22kr i A4-kiosken."],
+  ["Taktisk kont: ", "[tɑkʈɪsk kunt] Et falskt håp om at man kommer til å gjøre det bedre på eksamen etter to måneder på feriemodus."],
+  ["Kok: ", "[kuːk] Tradisjonell taktikk som anvendes for å komme opp til eksamen uten å kunne forskjellen på klasser og objekter."]
+];
+
+const funFactsPicker = () => {
+  return funFactList[Math.floor(Math.random() * (2 - 0 + 1) - 0)];
```

Comment:
Tror dette vil funke bedre:
```suggestion
  return funFactList[Math.floor(Math.random() * funFactList.length)];
```